src/pages/find-offers/FindOffers.jsx

```diff
@@ -29,6 +29,12 @@ const FindOffers = () => {
   const onBookmarkClick = (id) => {console.log(id)}
   const handleToggleOpenFilters = () => setOpenFilters(prev => !prev)
   const handleShowingTutorOffers = () => setShowingTutorOffers(prev => !prev)
+
+  const currentOffersOnPage = useMemo(() => {
```

Comment:
why do you need this here? your component should only map and show cards that are given to it